src/lib/comms/sol-netctl-impl-connman.c

```diff
@@ -424,6 +436,9 @@ get_services_properties(sd_bus_message *m, const char *path)
 
             r = sd_bus_message_exit_container(m);
             SOL_INT_CHECK(r, < 0, r);
+
+            if (!is_changed)
+                is_changed = true;
```

Comment:
minor, but you can just set the `is_changed = true` without checking for it... not a performance hit, will reduce number of lines.